docs-src/docs/reference/cli.md

```diff
@@ -658,6 +658,11 @@ Use the **create** subcommand to create a new load balancer.
   </div>
 </div>
 
+!!! Note "Ports and Health Checks"
+    When both the `--port` and the `--healthcheck-target` options are omitted, Layer0 configures the load balancer with some default values: `80:80/tcp` for ports and `TCP:80` for healthcheck target.
+    These default values together create a load balancer configured with a simple but functioning health check, opening up a set of ports that allows traffic to the target of the healthcheck.
```

Comment:
Consider adding a bit at the end of this line that spells out what each element of `80:80/tcp` means.